Bottom: 9312fb88fd80d3da765720948039bba6fca8caf9
Top:    5d9f75edb0026a357a65b40f6844dc59a4840761
Author: scott snyder <sss@karma>
Date:   2020-04-05 00:03:49 -0400

Refresh of changes-to-get-ctrl-alt-del

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
index 4916950..c93fd75 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Cyanide.java
@@ -47,7 +47,6 @@ public class Cyanide extends RandomIndexedComic {
 
 	@Override
 	protected int getPreviousStripId(BufferedReader br, String url) {
-                Log.d(Cyanide, "In getPreviousStripId, url = "+url);
 		int id = -1;
 		try {
 			String str;
@@ -117,7 +116,6 @@ public class Cyanide extends RandomIndexedComic {
 
 	@Override
 	protected String parse(String url, BufferedReader reader, Strip strip) throws IOException {
-                Log.d(Cyanide, "parse " + url);
 		boolean comic_found = true;
 		String str;
 		String final_str = null;
